osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -85,10 +95,86 @@ public override double Calculate(Dictionary<string, double> categoryRatings = nu
 
             return totalValue;
         }
-
+public double[] Main()
+    {
+        try
+        {   // Open the text file using a stream reader.
+            string tempstor = Path.GetTempPath();
+            string jdapath = tempstor + "jda.txt";
+            using (StreamReader sr = new StreamReader(jdapath))
+            {
+            // Read the stream to a string, and write the string to the console.
+                double[] errorval = {1,2,3};
+                double[] JumpDistanceArray = new double[32767];
+                string a = sr.ReadToEnd();
+                string[] JumpDistanceArray2 = a.Split(new char[] { ',' });
+                for (int i = 0; i < JumpDistanceArray2.Length; i++)
+                {
+                    double intermres = Convert.ToDouble(JumpDistanceArray2[i]);
+                    for (int m = 0; m < JumpDistanceArray2.Length; m++)
```

Comment:
this really doesnt make sense, you're overwriting the whole array with the last value?

Reply:
Now this one is quite the problem I've been facing, I tried doing recursive addition to arrays but it doesn't work ( It doesn't return arrays of data). Somehow when this function is called it doesn't add new values to my Array. Could you suggest a way to prevent an overwrite or seperate the new values with comma?

Reply:
By the way, the code you're referencing here should work as intended which is changing string into double[]. The problem is with the aim.cs issue you are referencing below.

Reply:
Fixed in #8 .